projects/adf-office-services-ext/src/lib/aos-extension.service.ts

```diff
@@ -112,22 +112,14 @@ export class AosEditOnlineService {
   ) {
     const protocolUrl = protocolHandler + ':ofe%7Cu%7C' + url;
 
-    const input = document.createElement('input');
-    const inputTop = document.body.scrollTop + 10;
-    input.setAttribute(
-      'style',
-      'z-index: 1000; background-color: rgba(0, 0, 0, 0); ' +
-        'border: none; outline: none; position: absolute; left: 10px; top: ' +
-        inputTop +
-        'px;'
-    );
-    document.getElementsByTagName('body')[0].appendChild(input);
-    input.focus();
-    location.href = protocolUrl;
+    const iframe = document.createElement('iframe');
+    iframe.style.display = 'none';
+    iframe.src = protocolUrl;
+
+    document.body.appendChild(iframe);
 
-    setTimeout(function() {
-      input.onblur = null;
-      input.remove();
+    setTimeout(() => {
+      document.body.removeChild(iframe);
```

Comment:
probably needs a null-check as `removeChild` raises errors for `null` and `undefined` values. the constant might be garbage collected before timer hits I think.